avocado/plugins/yaml_to_mux.py

```diff
@@ -293,10 +350,28 @@ def configure(self, parser):
             mux.add_argument("-m", "--mux-yaml", nargs='*', metavar="FILE",
                              help="Location of one or more Avocado"
                              " multiplex (.yaml) FILE(s) (order dependent)")
+            mux.add_argument('--mux-filter-only', nargs='*', default=[],
```

Comment:
I think it's OK to have filtering at the top most level (at the variants API) and cascading those to the individual implementations.  It's actually nice if we can have a neat model that accommodates all of them uniformly.